packages/@aws-cdk/aws-lambda-nodejs/lib/types.ts

```diff
@@ -66,6 +66,65 @@ export interface BundlingOptions {
    */
   readonly keepNames?: boolean;
 
+  /**
+   * Normally the esbuild automatically discovers `tsconfig.json` files and reads their contents during a build.
+   *
+   * However, you can also configure a custom `tsconfig.json` file to use instead.
+   *
+   * This can be useful if you need to do multiple builds of the same code with different settings:
+   *
+   * @default - tsconfig.json
+   */
+  readonly tsConfig? : string
```

Comment:
rename to `tsconfig` to be consistent with the types provided by `esbuild` (https://unpkg.com/browse/esbuild@0.8.23/lib/main.d.ts)

```suggestion
  readonly tsconfig? : string
```